incubator/hnc/go.sum

```diff
@@ -142,6 +142,7 @@ github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/pty v1.1.5/go.mod h1:9r2w37qlBe7rQ6e1fg1S/9xpWHSnaqNdHD3WcMdbPDA=
 github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
+github.com/kubernetes-sigs/multi-tenancy v0.0.0-20200221171633-2b5c81967af0 h1:JzZCwDzWZpZOMNgir26AiVSDuCWsJSGB2CX5pzT2/fM=
```

Comment:
This doesn't look right - this means it tried to download a package. Try reverting go.sum and re-running `make` to ensure it's not changed again.

Reply:
Reverted.